framework/src/org/checkerframework/framework/util/typeinference/DefaultTypeArgumentInference.java

```diff
@@ -212,14 +210,14 @@ protected AnnotatedTypeMirror getAssignedTo(ExpressionTree expression, Annotated
         Pair<InferenceResult, InferenceResult> argInference = inferFromArguments(typeFactory, afArgumentConstraints, targets);
 
         final InferenceResult fromArgEqualities = argInference.first;  //result 2.a
-        final InferenceResult fromArgSupertypes = argInference.second; //result 2.b
+        final InferenceResult fromArgSubandSupers = argInference.second; //result 2.b
```

Comment:
It is SubAndSuperTypes because those are the constraints used to create that result (rather than just Supers).  ClampToLowerBound remains unchanged.  That is, it still forces the inferred type to be at lowest the lower bound if it were inferred to be below the lower bound.